cdap-cli/src/main/java/co/cask/cdap/cli/command/schedule/UpdateTimeScheduleCommand.java

```diff
@@ -93,11 +87,11 @@ public void perform(Arguments arguments, PrintStream printStream) throws Excepti
 
   @Override
   public String getPattern() {
-    return String.format("update time schedule <%s> for <%s> <%s> " +
-                           "[description <%s>] at <%s> [properties <%s>] [constraints <%s>]",
-                         ArgumentName.SCHEDULE, ArgumentName.PROGRAM_TYPE, ArgumentName.PROGRAM,
-                         ArgumentName.DESCRIPTION, ArgumentName.CRON_EXPRESSION, ArgumentName.SCHEDULE_PROPERTIES,
-                         ArgumentName.SCHEDULE_RUN_CONSTRAINTS);
+    return String.format("update time schedule <%s> for workflow <%s> " +
```

Comment:
shouldn't this also accept a version? As in:
```
  start workflow <app-id.workflow-id> [version <app-version>] [<runtime-args>]
    Starts a workflow. '<runtime-args>' is specified in the format 'key1=a key2=b'.
```
and the same for the add schedule.